src/openzaak/components/zaken/api/serializers/zaken.py

```diff
@@ -564,7 +565,10 @@ def create(self, validated_data):
         # If it fails in any other way, we need to handle that by rolling back
         # the ZIO creation.
         try:
-            response = create_remote_oio(io_url, zaak_url)
+            if settings.CMIS_ENABLED:
+                oio = ObjectInformatieObject.objects.create(informatieobject=io_url, zaak=zaak_url, object_type='zaak')
+            else:
+                response = create_remote_oio(io_url, zaak_url)
```

Comment:
Same as mentioned earlier, to move the logic to `create_remote_oio`.

Reply:
Done